assets/js/script.js

```diff
@@ -19,21 +19,51 @@ var recipeHistoryEl = document.getElementById('recipe-history');
 //Event listener that runs search function on click
 
 var newSearch = document.getElementById('searchBtn');
-newSearch.addEventListener("click",fn1);
+newSearch.addEventListener("click", fn1);
 
 //Fetch list of recipe names from Spoonacular API based on ingredient inputs
 
+<<<<<<< HEAD
+function fn1(e) {
+
+
+    var ingred = document.getElementById('form1').value;
+    var ingred2 = document.getElementById('form2').value;
+    var ingred3 = document.getElementById('form3').value;
+    var allIngreds = ingred + ",+" + ingred2 + ",+" + ingred3;
+=======
```

Comment:
Hey Chris! Do we need these extra lines, ie 26-34? They look like a duplicate of the code that follows?

Reply:
I'm sure this is just a merge conflict resolution thing